### ISSUE TITLE
Sanity test cases

### ISSUE THREAD

**Issue by chrisvrose** [IEEE-RVCE/GateKeeper#16]
This is a set of test cases to ensure the following is done:
1. The application starts up.
2. The root API is accessible without any auth, returns 200.
3. The database can be created without any errors.

**Comment by ChiragJS** [comment]
Can I work on this issue ?

**Comment by chrisvrose** [comment]
Yes, would be good. Assigning

**Comment by chrisvrose** [comment]
I'll take some of the starter testcases.

**Comment by Prajwalprakash3722** [comment]
@ChiragJS what's the status on this?

**Comment by ChiragJS** [comment]
Yea , so as we had to get the backend stuff started , skipped writing the tests, but @chrisvrose did add some but had to remove it, because we were having some build issues

**Comment by chrisvrose** [comment]
Most likely from missing configuration. Will look into it over the couple
of weekends. If anyone else wants to look at it at present, please
feel free.

**Comment by Prajwalprakash3722** [comment]
Any luck on this ?

**Comment by chrisvrose** [comment]
Yup, I'm still working on it. Didn't get much chance since last weekend
since there's work going on, but I'll be pushing out small chunks of tests
over the weekend.
